pyproject.toml

```
@@ -0,0 +1,76 @@
+[build-system]
+requires = ["setuptools>=68.2.2"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "common_ci_utils"
+version = "0.1.1"
+authors = [
+  { name="IBM Storage QE", email="ocs-qe-ecosystem@redhat.com" },
+]
+description = "Place for common utils, functionality and libraries used in the testing and CI."
+readme = "README.md"
+requires-python = ">=3.9"
```

```suggestion
requires-python = ">=3.10"
```
accessing data files via namespace packages was added only in Python 3.10